runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -72,6 +72,7 @@
     RESET_LOGUNIT(47, new TypeToken<CorfuPayloadMsg<Long>>(){}, true, false),
     LOG_ADDRESS_SPACE_REQUEST(48, TypeToken.of(CorfuMsg.class)),
     LOG_ADDRESS_SPACE_RESPONSE(49, new TypeToken<CorfuPayloadMsg<StreamsAddressResponse>>(){}),
+    SUFFIX_TRIM(69, new TypeToken<CorfuPayloadMsg<Long>>() {}),
```

Comment:
I don't think this needs to be a payloaded message with an epoch. CorfuMsg already has epoch as a part of the base fields.